gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/linker/EntitiesLinker.java

```diff
@@ -17,4 +19,7 @@
     default void link(Bundle filteredBundle, Bundle bundle, File rootFolder) {
         link(filteredBundle, bundle);
     }
+
```

Comment:
Please provide method level comment to understand why we need this and when does it get called?